src/audio/general.c

```diff
@@ -2373,6 +2373,8 @@ char sBoolStrs[3][5] = { "OFF", "ON", "STBY" };
 u8 sAudioNatureFailed = false;
 u8 sPeakNumNotes = 0;
 
+#if OOT_DEBUG
+
 void AudioDebug_SetInput(void) {
     Input inputs[MAXCONTROLLERS];
```

Comment:
I never realized how much debug crap there was at the top of this file here. thoughts on doing a inc.c include and ifdeffing that out?
(also regardless of the answer to that question, you can remove the newline after the `#if` )

Reply:
Yeah that's a lot cleaner. I put the if-def in the `.inc` file, not sure what you prefer

Reply:
I was thinking from general.c so you can see either the debug stuff being included, or the retail stuff
```c
#if OOT_DEBUG
#include debug.inc.c
#else
void AudioDebug_Draw(GfxPrint* printer) {
}

void AudioDebug_ScrPrt(const char* str, u16 num) {
}
#endif